### PR TITLE
feat: add flag to skip resources in clear-account

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

**Review by efimk-lu — COMMENTED** [review]
@saartochner , not sure it will help you, one of the things you want to skip are specific cloudformation stacks

**Comment by saartochner** [comment]
@efimk-lu You are right, this feature does not let me skip specific stacks. My goal is to skip the cloudformation entirely :)

File: src/commands/clear-account.js [hunk 1]
```diff
@@ -271,6 +290,11 @@ ClearAccountCommand.flags = {
 	httpProxy: flags.string({
 		description: "URL of the http/https proxy (when running in a corporate network)",
 		required: false
+	}),
+	skipResources: flags.string({
+		description:
+			"List of resources to skip, separated with ','. Example: S3,IAMRoles",
```
Comment (efimk-lu) [inline src/commands/clear-account.js:296]: Add here a list of supported resource names

**Review by efimk-lu — APPROVED** [review]
See my comment

**Pull request closed without merging**